projects/storefrontlib/src/cms-components/navigation/search-box/search-box.component.html

```diff
@@ -40,6 +40,9 @@
   *ngIf="results$ | async as result"
   class="results"
   (click)="close($event, true)"
+  (mouseenter)="disableClose()"
+  (mouseleave)="disableClose(false)"
```

Comment:
Do we need the `mouseleave`? I got good results without it

Reply:
Good results = same results as with it

Reply:
It was actually not working great. The keyboarding was broking once you'd hover over the search box panel. I've taken a different approach, with the side effect that the search box doesn't blur at all when you only click on a search result. That can be seen as an improvement, but I can also imagine that once we have smarter focus when you land on a (pdp) page, it will change again - which is fine imho.